pandas/io/formats/style.py

```diff
@@ -1006,19 +1006,23 @@ def hide_columns(self, subset) -> "Styler":
     def _highlight_null(v, null_color: str) -> str:
         return f"background-color: {null_color}" if pd.isna(v) else ""
 
-    def highlight_null(self, null_color: str = "red") -> "Styler":
+    def highlight_null(self, null_color: str = "red", subset=None) -> "Styler":
         """
         Shade the background ``null_color`` for missing values.
 
         Parameters
         ----------
-        null_color : str
+        null_color : str, default 'red'
+        subset : IndexSlice, default None
```

Comment:
This isn't an IndexSlice right? Just a label or list of labels?

Reply:
It is an IndexSlice, since it calls `self.applymap` and `applymap` uses IndexSlice.

< code >

https://github.com/pandas-dev/pandas/blob/e22a65657c1c6f2bfe278d38690dfd9dd157edef/pandas/io/formats/style.py#L729-L736

< proof >

![image](https://user-images.githubusercontent.com/26521816/74340193-73db8200-4de0-11ea-8a0d-31169e4da660.png)

[< the doc >](https://pandas.pydata.org/pandas-docs/stable/user_guide/style.html#Finer-control:-slicing)

Reply:
The test case here though is not for an IndexSlice rather a list of labels. I think for a natural API should go with Label / Sequence of Labels as mentioned above

Reply:
It can take Label / Sequence of Labels as well, just like other Styler built-in methods.

![image](https://user-images.githubusercontent.com/26521816/74450496-1e26d880-4eb9-11ea-90d0-7c110f62ebd2.png)

Reply:
Can you update this to `label or list of labels`?

Reply:
sure.